src/rt/tracegc.d

```diff
@@ -104,14 +119,21 @@ private string generateTraceWrappers()
         static if (name.length > 3 && name[0..3] == "_d_")
         {
             mixin("alias Declaration = " ~ name ~ ";");
-            code ~= generateWrapper!Declaration();
+            code ~= generateWrapper!(Declaration, ParamPos.front)();
+        }
+        static if (name.length > 3 && name[0..3] == "gc_")
+        {
+            mixin("alias Declaration = " ~ name ~ ";");
+            code ~= generateWrapper!(Declaration, ParamPos.back)();
         }
     }
 
     return code;
 }
 
-private string generateWrapper(alias Declaration)()
+static enum ParamPos { front, back }
+
+private string generateWrapper(alias Declaration, ParamPos pos)()
```

Comment:
```suggestion
private string generateWrapper(alias Declaration, ParamPos pos = ParamPos.front)()
```

This will make the change at line 122 go away.

Reply:
Done